Visualization/RemoveDuplicates.sh

```diff
@@ -1,10 +1,26 @@
 #!/usr/bin/env bash
-# A simple script that removes multiple hits from BackBlast results.
-
-for csv;
-do
-	echo Removing duplicate hits from on ${csv}
-	sort -k 1,1 -t , -u ${csv} > "$csv.out"
-done
-echo Done.
+set -euo pipefail
```

Comment:
```set -euo``` change to ```set -exuo```. ```-x``` makes it so that bash will dump a stack trace.

Reply:
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin

Reply:
I've now added `set -x` below (so that it does not clog up the 'help' statement but only provides a stack trace once the script starts to run).